pkg/gce-pd-csi-driver/gce-pd-driver.go

```diff
@@ -52,6 +57,7 @@ func (gceDriver *GCEDriver) SetupGCEDriver(cloudProvider gce.GCECompute, mounter
 
 	gceDriver.name = name
 	gceDriver.nodeID = nodeID
+	gceDriver.vendorVersion = vendorVersion
```

Comment:
There's a place down in the cloud provider file where you're putting the driver name/version in the string.  You may want to plumb this down to there too